async-kafka/apps/emergency-location-service/src/main/java/com/redhat/training/emergency/Application.java

```diff
@@ -0,0 +1,32 @@
+package com.redhat.training.emergency;
+
+import java.util.Arrays;
+import java.util.TimeZone;
+
+import javax.annotation.PostConstruct;
+import javax.sql.DataSource;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.context.properties.ConfigurationProperties;
+import org.springframework.boot.jdbc.DataSourceBuilder;
+import org.springframework.context.ApplicationContext;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.stereotype.Component;
+
+@SpringBootApplication
+public class Application {
+
+	@PostConstruct
+	public void setTimezone() {
+		TimeZone.setDefault(TimeZone.getTimeZone("UTC"));
+	}
+
+	public static void main(String[] args) {
+		SpringApplication.run(Application.class, args);
+	}
+
+}
```

Comment:
Jaime also is very sensitive with this last lines too:D I guess it comes from python programming because I never remember being careful about this last line thing. Python PEP suggests one I know but is this a kinda of general startdart? do you know that?